woocommerce/woocommerce-gutenberg-products-block
# Add dynamic rendering + SSR to Hand-picked Products block

See #586 – this updates the Hand-picked Products block to use the new dynamic & ServerSideRender approach to display products.

### Screenshots

The CSS is still a little off for these blocks, but they render correctly:

![Screen Shot 2019-05-22 at 2 08 50 PM](https://user-images.githubusercontent.com/541093/58197869-24c24f80-7c9b-11e9-961d-d3601165caf5.png)

### How to test the changes in this Pull Request:

1. Add a Hand-picked Product block to a post before activating this PR
2. Build the PR
3. View your existing post, the block should still render correctly
4. Add a new Hand-picked Products block to a post, it should work correctly – show the right products, allow for selecting new products in the sidebar, toggling the content, etc.
5. View your post on the frontend, it should display same as the editor preview
